tasks/get_objects_for_app.yml

```diff
@@ -15,6 +15,7 @@
 - name: Set OpenShift objects to manage
   set_fact:
     builds: "{{ templates | map('regex_search', '.*/bc.*\\.yml\\.j2$') | select('string') | list }}"
+    configmaps: "{{ templates | map('regex_search', '.*/cm.*\\.yml\\.j2$') | select('string') | list }}"
```

Comment:
This is not specific to this regex but I don't understand why the regex is not `.*/cm_.*\\.yml\\.j2$` ? Is `cmb.yml.j2` a valid filename ?

Reply:
Yes, `cmb.yml.j2` is a valid name. I don't enforce the usage of a underscore to allow a simple `cm.yml.j2` as we do for a deployment configuration, _e.g._ `dc.yml.j2`.